### PR TITLE
OADP-1941: Default datamover SC and add mover annotation

### PR DESCRIPTION
**Pull request by eemcmullan**

### CODE REVIEW

File: controllers/datamover.go [hunk 1]
```diff
@@ -91,6 +90,17 @@ func (r *DPAReconciler) ReconcileDataMoverController(log logr.Logger) (bool, err
 
 			return false, err
 		}
+
+		adpNS := corev1.Namespace{}
+		if err := r.Get(r.Context, types.NamespacedName{Name: r.NamespacedName.Namespace}, &adpNS); err != nil {
+			return false, err
+		}
+
+		// add volsync privileged mover annotation
```
Comment (shubham-pampattiwar) [inline controllers/datamover.go:99]: +1

**Review by shubham-pampattiwar — APPROVED** [review]

**Review by weshayutin — APPROVED** [review]
data and file permissions are correct :) well done!
```
sh-4.4$ cd /var/lib/mysql/
sh-4.4$ ls -la
total 24
drwxrwsr-x. 4 root  mysql  4096 Jun  2 18:08 .
drwxr-xr-x. 1 root  root     53 Jun 20  2022 ..
drwxrws---. 6 mysql mysql  4096 Jun  2 18:08 data
drwxrws---. 2 root  mysql 16384 Jun  2 17:48 lost+found
srwxrwxrwx. 1 mysql mysql     0 Jun  2 18:08 mysql.sock
sh-4.4$ cat /etc/passwd | grep mysql
mysql:x:27:27:MySQL Server:/var/lib/mysql:/sbin/nologin
sh-4.4$ cat /etc/group | grep mysql
root:x:0:mysql
mysql:x:27:
sh-4.4$ 
```

**Comment by shubham-pampattiwar** [comment]
Tested B/R for the todolist mysql app, worked me as well !

**Comment by kaovilai** [comment]
[flake](https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_oadp-operator/1058/pull-ci-openshift-oadp-operator-master-4.11-operator-e2e-aws/1664709138594140160#1:build-log.txt%3A932)
```
"time=\"2023-06-02T20:14:59Z\" level=error msg=\"Timed out awaiting reconciliation of volumesnapshotcontent snapcontent-14546aa9-6f66-4d35-ac03-8dfc8fb7b2d1\" backup=openshift-adp/mongo-datamover-e2e-92baba90-0180-11ee-b18b-0a580a818460 cmd=/plugins/velero-plugin-for-vsm logSource=\"/opt/app-root/src/velero-plugin-for-vsm/internal/util/util.go:472\" pluginName=velero-plugin-for-vsm",
```
/retest

**Pull request merged**